util/dbterror/ddl_terror.go

```diff
@@ -391,4 +385,13 @@ var (
 
 	// ErrColumnInChange indicates there is modification on the column in parallel.
 	ErrColumnInChange = ClassDDL.NewStd(mysql.ErrColumnInChange)
+
+	// ErrAlterTiFlashModeForTableWithoutTiFlashReplica returns when set tiflash mode on table whose tiflash_replica is null or tiflash_replica_count = 0
+	ErrAlterTiFlashModeForTableWithoutTiFlashReplica = ClassDDL.NewStdErr(mysql.ErrUnsupportedDDLOperation, parser_mysql.Message("Warning: you are altering tiflash mode for the table whose tiflash replica count is zero. The tiflash mode will finally take effect when you set tiflash replica!", nil))
+
+	// ErrUnsupportedTiFlashOperationForSysTable means we don't support the alter tiflash related action(e.g. set tiflash mode, set tiflash replica) for system table.
+	ErrUnsupportedTiFlashOperationForSysTable = ClassDDL.NewStdErr(mysql.ErrUnsupportedDDLOperation, parser_mysql.Message("alter tiflash related actions for tables in system database is unsupported", nil))
```

Comment:
```suggestion
	ErrUnsupportedTiFlashOperationForSysTable = ClassDDL.NewStdErr(mysql.ErrUnsupportedDDLOperation, parser_mysql.Message("Cannot configure TiFlash settings for a system table", nil))
```

Reply:
Also, what about in-memory but not system tables?

Reply:
I update the error code name ~